polaris-react/src/components/Button/Button.scss

```diff
@@ -328,20 +328,13 @@
       top: 0.5px;
       bottom: 0.5px;
       border-radius: inherit;
-      // adjust shine for critical and success buttons
-      box-shadow: var(--p-shadow-button-primary-experimental);
```

Comment:
Shadow for critical and success primary buttons were combined so I've removed them here to add to the individual classes below.